Manifests/ManifestsApple/com.apple.mdm.plist

```diff
@@ -374,6 +374,52 @@
 			<key>pfm_type</key>
 			<string>boolean</string>
 		</dict>
+		<dict>
+			<key>pfm_ios_min</key>
+			<string>15</string>
+			<key>pfm_macos_min</key>
+			<string>12</string>
+			<key>pfm_name</key>
+			<string>AssignedManagedAppleID</string>
+			<key>pfm_title</key>
+			<string>Assigned Managed Apple ID</string>
+			<key>pfm_tvos_min</key>
+			<string>15</string>
+			<key>pfm_type</key>
+			<string>string</string>
+		</dict>
+		<dict>
+			<key>pfm_ios_min</key>
+			<string>15</string>
+			<key>pfm_macos_min</key>
+			<string>12</string>
+			<key>pfm_name</key>
+			<string>EnrollmentMode</string>
+			<key>pfm_range_list</key>
+			<array>
+				<string>BYOD</string>
+			</array>
+			<key>pfm_title</key>
+			<string>EnrollmentMode</string>
+			<key>pfm_tvos_min</key>
+			<string>15</string>
+			<key>pfm_type</key>
+			<string>string</string>
+		</dict>
+		<dict>
+			<key>pfm_ios_min</key>
+			<string>15</string>
+			<key>pfm_macos_min</key>
+			<string>12</string>
+			<key>pfm_name</key>
+			<string>RequiredAppIDForMDM</string>
+			<key>pfm_title</key>
+			<string>Required App ID for MDM</string>
+			<key>pfm_tvos_min</key>
+			<string>15</string>
+			<key>pfm_type</key>
+			<string>integer</string>
```

Comment:
Apple notably has no description on this in their documentation... so if there's something you know that would be helpful to add as a description, let's add it :)

Reply:
See 0a4c542.